plugins/modules/create_blueprint.py

```diff
@@ -122,6 +123,27 @@ def increment_version(version: str, version_type: str) -> str:
 
 
 def main() -> None:
+    try:
+        try:
+            import toml
+
+            HAS_TOML = True
+        except ImportError:
+            HAS_TOML = False
+
+        if not HAS_TOML:
+            try:
+                import pytoml as toml
+
+                HAS_TOML = True
+                self.toml = toml
+            except ImportError:
+                HAS_TOML = False
+    except Exception as e:
+        self.module.fail_json(
+            msg="Exception encountered during execution: %s" % to_text(e)
+        )
+
```

Comment:
We don't have to do this check here, we can use toml from the weldr class like this: `weldr.toml.dump(toml_data, fd)` on line 218

Reply:
thanks for the feedback, i've made these changes